mypy.ini

```diff
@@ -0,0 +1,6 @@
+[mypy]
+
+files = pystiche/core
```

Comment:
Right now we only check the core package. The remaining packages should be checked in separate PRs.